internal/compiler/wasm/wasm.go

```diff
@@ -111,6 +111,7 @@ var builtinsFunctions = map[string]string{
 	ast.JSONMarshal.Name:                "opa_json_marshal",
 	ast.JSONUnmarshal.Name:              "opa_json_unmarshal",
 	ast.ObjectFilter.Name:               "opa_object_filter",
+	ast.ObjectGet.Name:                  "opa_obj_get",
```

Comment:
[nit] If `object.filter` is `opa_object_filter`, shouldn't we use `opa_object_get` here? 🤔 I take it we already have an `opa_object_get` use for internal purposes; but then maybe we should rename filter to `opa_obj_filter`; or rename the internal function...

Reply:
How about `builtin_object_get`, `builtin_object_filter`, etc. This way we can easily tell whether it's a function that implements a built-in. I agree that `opa_obj_get` is suboptimal in terms of consistency...